go.mod

```diff
@@ -23,6 +23,9 @@ require (
 	gopkg.in/ini.v1 v1.67.0
 )
 
+// TODO: remove this once the keyring PR is merged upstream
+replace github.com/byteness/keyring => github.com/kennethwkz-mm/keyring v1.3.5-0.20250902093321-78c9afacce36
```

Comment:
We should remove this before merging this and replace once `keyring` changes are merged